src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -823,7 +825,48 @@ private async Task<string> CreatePullRequestAsync(List<UpdateAssetsParameters> u
                 var message = new StringBuilder();
                 await CalculateCommitMessage(coherencyUpdate.update, coherencyUpdate.deps, message);
                 await CalculatePRDescription(coherencyUpdate.update, coherencyUpdate.deps, description);
-                await darc.CommitUpdatesAsync(targetRepository, newBranchName, coherencyUpdate.deps.Select(du => du.To).ToList(), message.ToString());
+
+                List<GitFile> commitedFiles = await darc.CommitUpdatesAsync(targetRepository, newBranchName, coherencyUpdate.deps.Select(du => du.To).ToList(), message.ToString());
+
+                UpdatePRDescriptionDueConfigFiles(commitedFiles, description);
+            }
+        }
+
+        private void UpdatePRDescriptionDueConfigFiles(List<GitFile> commitedFiles, StringBuilder description)
+        {
+            GitFile globalJsonFile = commitedFiles.
+                Where(gf => gf.FilePath.Equals("global.json", StringComparison.OrdinalIgnoreCase)).
+                FirstOrDefault();
+
+            if (globalJsonFile != null)
+            {
+                bool hasSdkVersionUpdate = globalJsonFile.Metadata.ContainsKey(GitFileMetadataName.SdkVersionUpdate);
+                bool hasToolsDotnetUpdate = globalJsonFile.Metadata.ContainsKey(GitFileMetadataName.ToolsDotNetUpdate);
+
+                string globalJsonSectionStartMarker = $"[marker]: <> (Begin:GlobalJson Updates)";
+                string globalJsonSectionEndMarker = $"[marker]: <> (End:GlobalJson Updates)";
+                int sectionStartIndex = RemovePRDescriptionSection(globalJsonSectionStartMarker, globalJsonSectionEndMarker, ref description);
+                StringBuilder globalJsonSection = new StringBuilder();
+
+                globalJsonSection.AppendLine(globalJsonSectionStartMarker);
+                globalJsonSection.AppendLine("- **Updates to global.json:**");
+                globalJsonSection.AppendLine();
+
+                if (hasSdkVersionUpdate)
+                {
+                    globalJsonSection.AppendLine($"  - Updates the version of .NET SDK used to " +
+                        $"{globalJsonFile.Metadata[GitFileMetadataName.SdkVersionUpdate]}");
+                }
+
+                if (hasToolsDotnetUpdate)
+                {
+                    globalJsonSection.AppendLine($"  - Updates the version of .NET Runtime used to " +
```

Comment:
I'm fuzzy in what the difference is between tools and sdk nodes, but I believe this also points to an SDK and not to a runtime?

Reply:
I checked with Tomas and ChrisC and it's indeed something specific to our infra and it's also the SDK version.